### DISCUSSION TITLE
How to get Goanimate

### DISCUSSION THREAD

**Discussion by Trainstream999** [GoAnimate-Wrapper/GoAnimate-Wrapper#112] [General]
How do I get Goanimate.

**Comment by Windows81** [comment]
Hi.  I recommend searching YouTube for your answer.  I stopped working on GoAnimate Wrapper over two years ago.  Things have changed a lot since then.

**Reply by IAmNotSiob** [reply]
Yeah, but you still need to answer him about how to get the IP.

**Reply by Windows81** [reply]
I haven't caught up.  Are you in the relevant Discord.com spaces?
